lib/js-file.js

```diff
@@ -191,6 +191,24 @@ JsFile.prototype = {
     },
 
     /**
+     * Returns the first token for the file from the AST.
+     *
+     * @returns {Object}
+     */
+    getFirstFileToken: function() {
```

Comment:
One last nit. This should be called `getFirstToken` instead, because the `File` is redundant. Same goes for `getLastToken`.